web/src/app/alerts/components/AlertDetails.js

```diff
@@ -423,8 +424,18 @@ function AlertDetails(props) {
 
   const { data: alert } = props
 
+  const notices = alert.pendingNotifications.map((n) => ({
+    type: 'WARNING',
+    message: `Notification Pending for ${n.destination}`,
+    details:
+      'One or more notifications are waiting to be sent. Contact methods may be rate limited on a per-service basis.',
+  }))
+
   return (
-    <Grid container spacing={2}>
+    <Grid container spacing={2} justify='center'>
+      <Grid item xs={12} lg={9}>
```

Comment:
It doesn't look like `lg={9}` is doing anything here

```suggestion
      <Grid item xs={12}>
```

Reply:
Are notices being rendered outside of our App's container? I feel like this component should adhere to the same rules we apply to all other grid items here. Setting to `9` feels like an arbitrary value and could lead to it being misaligned in the future, since the other grid items are all set to `12`.

Reply:
So it looks like it's happening because we're nesting the notification panel inside of a grid, which is already a grid- I've come across layout issues like this in the past when you have multiple grids trying to work together. I can do some investigating on this today. We shouldn't need to use `lg={9}` when the other cards do not